pages/common/routers/form.js

```diff
@@ -27,6 +28,22 @@ if (process.env.CSRF === 'false') {
   _checkSecret = defaultMiddleware;
 }
 
+const getOptionReveals = (schema, values) => {
+  return Object.keys(schema).reduce((obj, key) => {
+    const field = schema[key];
+    if (!field.options) {
+      return obj;
+    }
+    const selectedOptions = field.options.filter(opt => castArray(values[key]).includes(opt.value));
+    return selectedOptions.reduce((o, opt) => {
+      return {
+        ...o,
+        ...(opt.reveal || {})
+      };
+    }, {});
```

Comment:
There's no mention of `obj` here, which makes me suspect that this won't work if there's more than one field in the schema which has reveal options.

I _think_ it needs this:

```suggestion
    }, obj);
```

Reply:
Narrator voice: It did need that.